packages/css/.storybook/preview-head.html

```diff
@@ -0,0 +1,104 @@
+<style>
+  /* Box sizing rules */
+  *,
+  *::before,
+  *::after {
+    box-sizing: border-box;
+  }
```

Comment:
This should be extracted to `reset.css`

Reply:
That's only for the storybook preview right now...